files/zh-cn/web/html/block-level_elements/index.html

```diff
@@ -42,7 +42,7 @@ <h2 id="块级元素与行内元素">块级元素与行内元素</h2>
  <dd>一般块级元素可以包含行内元素和其他块级元素。这种结构上的包含继承区别可以使块级元素创建比行内元素更”大型“的结构。</dd>
 </dl>
 
-<p>HTML 标准中块级元素和行内元素的区别至高出现在 4.01 标准中。在 HTML5，这种区别被一个更复杂的<a href="/zh-CN/docs/HTML/Content_categories" title="/zh-CN/docs/HTML/Content_categories">内容类别</a>代替。”块级“类别大致相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Flow_content" title="/zh-CN/docs/HTML/Content_categories#Flow_content">流内容</a>类别，而”行内“类别相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content" title="/zh-CN/docs/HTML/Content_categories#Phrasing_content">措辞内容</a>类别，不过除了这两个还有其他类别。</p>
+<p>HTML 标准中块级元素和行内元素的区别至高出现在 4.01 标准中。在 HTML5，这种区别被一个更复杂的<a href="/zh-CN/docs/HTML/Content_categories">内容类别</a>代替。”块级“类别大致相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Flow_content">流内容</a>类别，而”行内“类别相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content">措辞内容</a>类别，不过除了这两个还有其他类别。</p>
```

Comment:
```suggestion
<p>HTML 标准中块级元素和行内元素的区别至高出现在 4.01 标准中。在 HTML5，这种区别被一个更复杂的<a href="/zh-CN/docs/HTML/Content_categories">内容类别</a>代替。”块级“类别大致相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#流内容">流内容</a>类别，而”行内“类别相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#措辞内容">措辞内容</a>类别，不过除了这两个还有其他类别。</p>
```

@yin1999 这个页面目前还没有中文翻译，要修改锚文本吗？

Reply:
not necessary